pylint/extensions/broad_try_clause.py

```diff
@@ -59,7 +60,7 @@ def _count_statements(self, try_node):
 
         return statement_count
 
-    def visit_tryexcept(self, node: nodes.TryExcept) -> None:
+    def visit_tryexcept(self, node: Union[nodes.TryExcept, nodes.TryFinally]) -> None:
```

Comment:
Really? That might have an effect in other places as well. When I added this typing I assumed the method name reflects the node being visited..

Reply:
Never mind, I just saw how this checker does this.